src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -325,10 +546,13 @@ public String autoConvertKnownFile(UploadedAutoconvertFile file, HttpServletRequ
      * @param result The BindingResult object for errors.
      * @return A String of the local file name for the ASCII file.
      */
+    /*
     @RequestMapping(value = "/parse", method = RequestMethod.POST)
     @ResponseBody
     public String parseFile(AsciiFile file, BindingResult result) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)